controllers/item.py

```diff
@@ -9,14 +11,22 @@ def index():
     get_items = Item.query.all()
     item_schema = ItemSchema(many=True)
     items = item_schema.dump(get_items)
-    return make_response(jsonify({"item": items}))
+    item_entities = []
+
+    for item in items:
+        print(item)
+        item_entity = ItemEntity(item)
+        item_entities.append(item_entity)
+        
+    return make_response(jsonify({"item": [i.toJSON() for i in item_entities]}))
 
 # GET Method / READ one item by ID
 def get_by_id(item_id):
     get_item = Item.query.get(item_id)
     item_schema = ItemSchema()
     item = item_schema.dump(get_item)
-    return make_response(jsonify({"item": item}))
+    item_entity = ItemEntity(item)
+    return make_response(jsonify({"item": item_entity}))
```

Comment:
I think you need to change to
```suggestion
    return make_response(jsonify({"item": item_entity.toJSON()}))
```

Reply:
Yes!